### PR TITLE
Space Travelers' Hub

### PR DESCRIPTION
**Pull request by Amalcxc**
###  All the project requirements.

- React components
- React Router
- Connect React and Redux
- Handle events in a React app
-  unit tests with React Testing Library
- styles in a React app
- React life cycle methods
- store, actions, and reducers in React

### CODE REVIEW

File: src/tests/rocketsProfile.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import store from '../redux/configureStore';
+
+test('matches RocketsProfile component snapshot', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <RocketsProfile />
+    </Provider>,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (Lameck1) [inline src/tests/rocketsProfile.test.js:13]: - [ ] In all tests you have only implemented snapshot tests. Snapshot tests only assert that the components are rendering correctly. The functionality of the app still needs to be tested. Please write some more tests that test the real functionalities of the app. For example:

1. Test that asserts that Rockets are loaded on the home page
2. Test that asserts that Missions are loaded on the missions page
3. Test that asserts that a user can click on the mission link and the app navigates to the missions page
4. Test that asserts that a user can join a mission
5. Test that asserts that a user can leave a mission
6. Test that asserts that a user can book a rocket
7. Test that asserts that a user can cancel rocket booking
8. Test that asserts that a user can click on the My Profile link and the app navigates to the My Profile page

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      <div className="col-4 pt-3">
+        <NavLink to="/" activeClassName="active-link ">
+          <span className="rockets-nav px-3 mx-3">Rockets</span>
+        </NavLink>
+        <NavLink to="/missions" activeClassName="active-link">
+          <span className="missions-nav px-3 mx-3">Missions</span>
+        </NavLink>
+        <NavLink to="/Profile" activeClassName="active-link">
+          <span className="profile-nav px-5 ">Profile</span>
+        </NavLink>
```
Comment (Lameck1) [inline src/components/Header.js:22]: - [ ] This requirement is half-implemented. We are supposed to show which page the user is currently on by underlining the active link. Please take another look at that specific requirement and make the needed adjustment 👇🏼:
---
> Create routes and view components (Rockets, Missions, My Profile, Dragons [only if your team has 3 members]). Use <NavLink /> for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link).

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @Amalcxc & @Rocio01, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Amalcxc @Rocio01 , 

Great work implementing the changes by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**